ui/raidboss/data/06-ew/trial/zeromus-ex.ts

```diff
@@ -89,11 +89,15 @@ const triggerSet: TriggerSet<Data> = {
       type: 'GainsEffect',
       netRegex: { effectId: '6E9', capture: false },
       suppressSeconds: 5,
-      alertText: (_data, _matches, output) => output.text!(),
+      alertText: (_data, _matches, output) => {
+        if (_data.role === 'healer')
```

Comment:
Sorry, I disagree a little here. A lot of players have self-heals that they can use here to help. There's also not anything else going on, so it's not like this is particularly noisy.

Reply:
You're right, I will revert it